src/main/java/app/bpartners/geojobs/service/event/AnnotationModelDeliveryRequestedService.java

```diff
@@ -0,0 +1,137 @@
+      log.warn(
+          "No potential "
+              + event.getAnnotationModelDeliveryType()
+              + " objects found from ZDJ(id="
+              + zoneDetectionJobId
+              + ")");
+    } else {
+      var jobName =
+          getJobName(
+              humanZDJ, savedHumanJob, machineDetectedTiles, minimumConfidenceForDelivery, event);
+      eventProducer.accept(List.of(new HumanDetectionJobCreated(savedHumanJob.getId(), jobName)));
+    }
+  }
+
+  @NonNull
+  private String getJobName(
+      ZoneDetectionJob humanZDJ,
+      HumanDetectionJob savedHumanJob,
+      List<MachineDetectedTile> machineDetectedTiles,
+      Double minimumConfidenceForDelivery,
+      T event) {
+    LocalDateTime dateTime = LocalDateTime.ofInstant(now(), ZoneId.of("Europe/Paris"));
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm:ss");
+    switch (event.getAnnotationModelDeliveryType()) {
+      case TRUE_POSITIVE -> {
+        return humanZDJ.getZoneName()
+            + " - "
+            + savedHumanJob.getDetectableObjectConfigurations().stream()
+                .map(DetectableObjectConfiguration::getObjectType)
+                .toList()
+            + " "
+            + machineDetectedTiles.size()
+            + " images with object detection confidence >= "
+            + minimumConfidenceForDelivery * 100
+            + "%"
+            + " "
+            + dateTime.format(formatter);
+      }
+      case FALSE_POSITIVE -> {
+        return humanZDJ.getZoneName()
+            + " - "
+            + savedHumanJob.getDetectableObjectConfigurations().stream()
+                .map(DetectableObjectConfiguration::getObjectType)
+                .toList()
+            + " "
+            + machineDetectedTiles.size()
+            + " images with object detection confidence < "
+            + minimumConfidenceForDelivery * 100
+            + "%"
+            + " "
+            + dateTime.format(formatter);
+      }
+      case WITHOUT_DETECTED_OBJECT -> {
+        return humanZDJ.getZoneName()
+            + " - "
+            + savedHumanJob.getDetectableObjectConfigurations().stream()
+                .map(DetectableObjectConfiguration::getObjectType)
+                .toList()
+            + " "
+            + machineDetectedTiles.size()
+            + " images without object "
+            + minimumConfidenceForDelivery * 100
+            + " "
+            + dateTime.format(formatter);
+      }
+    }
+    throw new IllegalArgumentException(
+        "Unknown AnnotationModelDeliveryType " + event.getAnnotationModelDeliveryType());
+  }
+
+  private List<MachineDetectedTile> getMachineDetectedTiles(T event, String zoneDetectionJobId) {
+    boolean isGreater = true;
+    boolean isNotGreater = !isGreater;
+    return switch (event.getAnnotationModelDeliveryType()) {
+      case TRUE_POSITIVE ->
+          detectedTileRepository.findAllInDoubtByZdjJobIdGreaterThan(zoneDetectionJobId, isGreater);
```

Comment:
Instead, get all in doubt objects from database, then compare the computed confidence with the given minConfidenceForDelivery. 

Rename case when isGreater directly to GreaterThanOrEquals.